.github/workflows/build.yml

```diff
@@ -5,7 +5,6 @@ jobs:
   build:
     name: Build and Test
     runs-on: ubuntu-latest
-    if: "!contains(github.event.head_commit.message, '[ci skip]') && !contains(github.event.head_commit.message, '[skip ci]')"
```

Comment:
Why?

Reply:
Because this is already integrated into github actions (https://github.blog/changelog/2021-02-08-github-actions-skip-pull-request-and-push-workflows-with-skip-ci/)

Reply:
Since 2021? Why didn't I know?